lib/engines/content_object_store/app/controllers/content_object_store/content_block/documents_controller.rb

```diff
@@ -7,12 +7,10 @@ def show
     @content_block_document = ContentObjectStore::ContentBlock::Document.find(params[:id])
     @content_block_versions = @content_block_document.versions
 
-    linked_item_service = ContentObjectStore::GetLinkedContentItems.new(
+    embedded_content = ContentObjectStore::GetEmbeddedContentItems.new(
```

Comment:
should this actually be GetHostContentItems? we're returning the documents that host the embedded content if I've got it right?

Reply:
Woah yes this needs some work, I've glossed over the naming after looking at them for so long.

We certainly get back `host_content` and the objects and table etc should be named accordingly - shall fix. 

I'm less sure what we do about this service. `GetHostContentItems` conveys what's returned but doesn't speak to how it works. How about `GetHostContentItemsByTarget`?

Reply:
> We could add a method to help us: GetContentItems.by_embedded_document(doc)?

yep I think that works 👍